pkg/reconciler/ingress/ingress_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	clientgotesting "k8s.io/client-go/testing"
+	"k8s.io/utils/pointer"
```

Comment:
nit:` knative/pkg/ptr`

Reply:
Why prefer `knative.dev` packages to the upstream k8s ones?